build.assets/versions.mk

```diff
@@ -7,10 +7,7 @@ GOLANG_VERSION ?= go1.24.1
 GOLANGCI_LINT_VERSION ?= v1.64.7
 
 # NOTE: Remember to update engines.node in package.json to match the major version.
-# TODO(ravicious): When updating Node.js, see if corepack distributed with the new Node.JS version
-# is >= 0.31.0. If so, remove manual calls to install corepack@0.31.0 from CI scripts and
-# COREPACK_INTEGRITY_KEYS env var from windows/build.ps1.
-NODE_VERSION ?= 20.18.0
+NODE_VERSION ?= 22.14.0
```

Comment:
We should also check if the Web UI builds correctly after the update and if Storybook still works. 😅

Reply:
Tested, works fine.